.travis.yml

```diff
@@ -1,3 +1,9 @@
 language: node_js
 node_js: 
-  - "node"
\ No newline at end of file
+  - "node"
+env: 
+  global: 
+    - BROWSER_DRIVER_INSTALLER_CHROME_VERSION=67
```

Comment:
ideally, we could have an alias for the latest version, so we don't need to hard code this here (assuming, that is, this is possible) and we wouldn't have to change the versions here also

Reply:
I've created #3 and #4